install_config/cluster_metrics.adoc

```diff
@@ -99,6 +99,12 @@ for:
 The xref:metrics-deployer[Metrics Deployer] will be discussed in a later step,
 but you must first set up a service account for it:
 
+
+.Grant view permissions to the Hawkular Metrics service:
```

Comment:
And a `+` on the next line to join the command block to this step, if that's what you want.